### ISSUE TITLE
Weird static glitch over my screen HL2

### ISSUE THREAD

**Issue by Jerzyna82** [ValveSoftware/Source-1-Games#6904]
![Screenshot_20241226_151145_Gallery](https://github.com/user-attachments/assets/20f586df-0000-4e7b-aed2-9b2018304951)
![Screenshot_20241226_151139_Gallery](https://github.com/user-attachments/assets/a3b42d73-cafd-4854-86eb-52630fd1c199)
![Screenshot_20241226_151127_Gallery](https://github.com/user-attachments/assets/3b01d012-f3c2-4d1d-add2-d73e07f41e3b)
Reinstalling the game didnt help. I tried reseting game data, but the files came back (i turned off steam cloud first but it turned itself back on).
The issue is also in the menu.

**Comment by megakarlach** [comment]
This looks like GPU issues or something, have you tried to update the drivers?

**Comment by Jerzyna82** [comment]
I will check if its updated later, since im away from home. I will let you know tommorow.

**Comment by Jerzyna82** [comment]
Yeah problem solved, i cant belive i was so stupid. Thanks a lot for reminding me lol
